app/views/enterprises/shop.html.haml

```diff
@@ -20,10 +20,6 @@
   - content_for :order_cycle_form do
     = render partial: "change_order_cycle"
 
-  - content_for :ordercycle_sidebar do
-    .show-for-large-up.large-4.columns
-      = render partial: "shopping_shared/order_cycles"
-
```

Comment:
Not sure of this one, and specially about this line : 
`- content_for :ordercycle_sidebar do`

Reply:
The `content_for` helper stores content in a variable to be rendered by another layout file. It's rendered here:
https://github.com/openfoodfoundation/openfoodnetwork/blob/aca19edd3b49fe333f50d2219e0f1a4e30ac377c/app/views/shopping_shared/_header.html.haml#L13

There are still two other places supplying some OC information there. So I think that you did everything right.